pandas/core/indexes/base.py

```diff
@@ -695,13 +695,31 @@ def memory_usage(self, deep=False):
     # ops compat
     @deprecate_kwarg(old_arg_name='n', new_arg_name='repeats')
     def repeat(self, repeats, *args, **kwargs):
-        """
-        Repeat elements of an Index. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        """Repeat elements of an Index.
+
+        Parameters
+        ----------
+        repeats : int
+            The number of repetitions for each element.
+
+        Returns
+        -------
+        pandas.Index
+            Newly created Index with repeated elements.
 
         See also
```

Comment:
If I'm not wrong, "See Also" should have capital "A"

Reply:
Yes, that is correct (although that numpydoc will still recognize it correctly, as it tries to capitalize the words if they are not)